test/client/js/libs/handlebars/ircmessageparser/parseStyle.js

```diff
@@ -104,12 +109,70 @@ describe("parseStyle", () => {
 		expect(actual).to.deep.equal(expected);
 	});
 
+	it("should parse hex colors", () => {
+		const input = "test \x04FFFFFFnice \x02\x04RES006 \x0303,04\x04ff00FFcolored";
```

Comment:
is hex formatting foreground only?
also more color variation would be nice (see issue with pattern)